### ISSUE TITLE
Cannot install for Python 3.11

### ISSUE THREAD

**Issue by bryevdv** [kalekundert/autoclasstoc#34]
```
dev-311 ❯ pip install -U autoclasstoc==1.5
ERROR: Ignored the following versions that require a different python version: 1.5.0 Requires-Python >=3.6,<3.11; 1.5.1 Requires-Python >=3.6,<3.11; 1.5.2 Requires-Python >=3.6,<3.11
ERROR: Could not find a version that satisfies the requirement autoclasstoc==1.5 (from versions: 0.0.0, 1.0.1, 1.1.0, 1.1.1, 1.2.0, 1.2.1, 1.3.0, 1.4.0)
ERROR: No matching distribution found for autoclasstoc==1.5
```

**Comment by kalekundert** [comment]
Thanks for reporting this, and sorry for the slow response.  This was fixed by #25, but I haven't yet made a release that includes the fix.  I anticipate making a new release in the next week or so, though.

**Comment by kalekundert** [comment]
I just made a new release, so this issue should be resolved now.  Feel free to reopen if that isn't the case.
